### DISCUSSION TITLE
should i ditch vicuna 13b for wizardlm 7b?

### DISCUSSION THREAD

**Discussion by yesbroc** [oobabooga/text-generation-webui#1645] [Q&A]
im stuck with this decision since i dont want to keep both of them, yet i cant decide which is better. vicuna has better logical reasoning but wizardlm is way faster and beats vicuna on some instances in detail and creativity. im stumped, any ideas? p.s wizardlm also shows memory loss in my uses

**Comment by johndpope** [comment]
yes.

**Reply by yesbroc** [reply]
my wizardlm has memory loss
